android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java

```diff
@@ -379,11 +381,21 @@ private void processProperty(String name, Object value)
 
 			// Set list sections.
 			setSections((Object[]) value);
-		}
 
-		if (name.equals(TiC.PROPERTY_EDITING)) {
+		} else if (name.equals(TiC.PROPERTY_EDITING)) {
+			final TiViewProxy parent = getParent();
+
+			if (parent != null) {
+
+				// Due to Android limitations, selection trackers cannot be removed.
+				// Re-create ListView with new selection tracker.
+				releaseViews();
+				parent.add(this);
+			}
+
+		} else if (name.equals(TiC.PROPERTY_SHOW_SELECTION_CHECK)) {
 
-			// Update list.
+			// Update and refresh list.
 			update();
 		}
 	}
```

Comment:
I think we "need" to add support for the [selectedItems](https://titaniumsdk.com/api/titanium/ui/listview.html#selecteditems) property so that the app can fetch the currently selected items without an event listener.
```java
@Kroll.getProperty
public KrollDict[] getSelectedItems()
{
	// Return same array as 'itemsselected' event.
}
```

We have an iOS unit-test for the above, but we enable it for Android until we also add support for the [selectItems()](https://titaniumsdk.com/api/titanium/ui/listview.html#selectitem) method... and if do that then we should also support the [deselectItems()](https://titaniumsdk.com/api/titanium/ui/listview.html#deselectitem). _(sorry)_
https://github.com/appcelerator/titanium_mobile/blob/master/tests/Resources/ti.ui.listview.test.js#L239